.gitmodules

```diff
@@ -47,3 +47,9 @@
 [submodule "3rdparty/CEGUI/cegui"]
 	path = 3rdparty/CEGUI/cegui
 	url = git@gitlab.com:InnerPieceOSS/cegui.git
+[submodule "ext/CEGUI/portable-file-dialogs"]
+	path = ext/CEGUI/portable-file-dialogs
+	url = git@github.com:samhocevar/portable-file-dialogs.git
+[submodule "ext/CEGUI/stb"]
```

Comment:
if we're not gonna keep something around, don't make it a submodule... submodules are hard to get rid of later on

Reply:
submodule will introduce much less changes in your git history than directly put the code files here.